plugins/vendor/zoom/lib/model/kZoomEngine.php

```diff
@@ -353,11 +353,18 @@ protected function createEntryDescriptionFromMeeting($meeting)
 	 * @param entry $entry
 	 * @throws kCoreException
 	 */
-	protected function setEntryCategory($zoomIntegration, $entry)
+	protected function createCategoryEntry($zoomIntegration, $entry)
 	{
 		if ($zoomIntegration->getZoomCategory())
 		{
-			$entry->setCategories($zoomIntegration->getZoomCategory());
+			$categoryEntry = new categoryEntry();
+			$categoryEntry->setEntryId($entry);
+			$categoryEntry->setCategoryId($zoomIntegration->getZoomCategoryId());
+			$categoryEntry->setPartnerId($entry->getPartnerId());
+			$categoryEntry->setCreatorKuserId($entry->getKuserId());
+			$categoryEntry->setStatus(CategoryEntryStatus::ACTIVE);
+			$categoryEntry->save();
+			KalturaLog::info("Entry Id '" . $entry->getId() ."' added to Zoom category Id " . $zoomIntegration->getZoomCategoryId());
```

Comment:
If a string does not process values , don't use " prefer single quote '

Reply:
Changed to single quotes